src/modm/board/nucleo_f303re/module.lb

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016-2018, Niklas Hauser
+# Copyright (c) 2017, Fabian Greif
```

Comment:
...

Reply:
You added some original work here, so proudly add your name 🤓.